### PR TITLE
Update include-js to add attributes to script tag

### PR DESCRIPTION
**Pull request by jaybarra**
Updates include-js to allow passing in other script element attributes.

### CODE REVIEW

**Comment by frou** [comment]
Kinda nice that since this does a map merge, it can also do an upgrade of `:type` to `"module"` (to enable so-called JS Strict-Mode etc).

File: src/hiccup/page.clj [hunk 1]
```diff
@@ -78,7 +78,11 @@
   "Include a list of external javascript files."
   [& scripts]
   (for [script scripts]
-    [:script {:type "text/javascript", :src (util/to-uri script)}]))
+    (if (:src script)
```
Comment (weavejester) [inline src/hiccup/page.clj:81]: Would a `map?` be better here?

Comment (jaybarra) [reply src/hiccup/page.clj:81]: As long as there are no concerns about possibly `nil` being fed to the uri converter, in which case I can make it `if-let`

Comment (weavejester) [reply src/hiccup/page.clj]: If `nil` is passed, it's probably better to throw an error than to silently ignore it.

**Pull request closed without merging**